README.md

```diff
@@ -158,8 +165,191 @@ Note that the module does not support [NTLMSSP](http://en.wikipedia.org/wiki/NTL
 in Negotiate. NTLM, both v1 and v2, is an exploitable protocol and should be avoided
 where possible.
 
+Ubuntu 12.04 LTS packages for the latest version of Nginx compiled with the SPNEGO module
+are [available in this PPA](https://launchpad.net/~bcandrea/+archive/nginx-stable).
+
+
+Crash course to Windows KDC Configuration
```

Comment:
Split this out into a separate file and link it from the readme? Otherwise, lgtm.